Sources/GryphonLib/Utilities.swift

```diff
@@ -308,3 +308,21 @@ extension List {
 		return result
 	}
 }
+
+extension String {
```

Comment:
Having the extension in this file is a problem for the BootstrappingTest. There is a `Utilities.swift` file and a separate `Utilities.kt` file (inside the `Bootstrap` folder), and the code inside them should be the same (in each language). This means if you want to keep this code here, you'll have to write a Kotlin version of it in `Utilities.kt` for the BootstrappingTest to work.

Or, if you prefer, you can move this code to a file that is translated automatically, like `SharedUtilities.swift` or `Extensions.swift`.

Reply:
my bad, I missed it.